runners/sbt/src/main/scala/stryker4s/sbt/Stryker4sPlugin.scala

```diff
@@ -20,17 +22,38 @@ object Stryker4sPlugin extends AutoPlugin {
   import autoImport._
 
   override lazy val projectSettings: Seq[Def.Setting[_]] = Seq(
-    stryker := strykerTask.value
+    stryker := strykerTask.value,
+    logLevel in stryker := Level.Info,
+    onLoadMessage := "", // Prevents "[info] Set current project to ..." inbetween mutations
   )
 
   val strykerTask = Def.task {
-    // Run Stryker
+    // Run Stryker4s
     val currentState = state.value
+
+    // Set the log level used for stryker log messages
+    setStrykerLogLevel((logLevel in stryker).value)
+
     val result = new Stryker4sSbtRunner(currentState).run()
 
     result match {
       case ErrorStatus => throw new MessageOnlyException("Mutation score is below configured threshold")
       case _           => currentState
     }
   }
+
+  private def sbtLogLevelToLog4j(level: Level.Value): org.apache.logging.log4j.Level = {
```

Comment:
Minor nitpick, but IMO it's nicer to remove the braces on a function if it is only a pattern match